app/models/user.rb

```diff
@@ -8,4 +9,11 @@ class User < ActiveRecord::Base
   validates :email, uniqueness: true
```

Comment:
@pg20 about uniqueness - please add `uniqueness: { case_sensitive: false }` so our app will treat "u@email.com" and "U@EMAIL.COM" as same string :)
and I see you added index on email  - that together with this improved validation in modelwill be enough to ensure we don't have 2 same users in our datatabse :)